packages/client/src/api/fetcher.ts

```diff
@@ -2,7 +2,13 @@ import { VERSION } from '../version';
 import { FetcherError, PossibleErrors } from './errors';
 
 const resolveUrl = (url: string, queryParams: Record<string, any> = {}, pathParams: Record<string, string> = {}) => {
-  const query = new URLSearchParams(queryParams).toString();
+  // Remove nulls and undefineds from query params
+  const cleanQueryParams = Object.entries(queryParams).reduce((acc, [key, value]) => {
+    if (value === undefined || value === null) return acc;
```

Comment:
```suggestion
    if (value == null) return acc;
```

Nitpick.

Reply:
I tend to be explicit in this particular case and try to avoid coercion. I guess we could have a helper isNullish or something like that to be less verbose everywhere.

Reply:
👍  sounds good. Yeah, a helper would be nice, to make it less verbose, like you said. But any way works for me.